cmd/nexusUpload_test.go

```diff
@@ -48,6 +51,9 @@ func (m *mockUtilsBundle) getExecRunner() execRunner {
 }
 
 func (m *mockUtilsBundle) setProperty(pomFile, expression, value string) {
+	pomFile = strings.ReplaceAll(pomFile, "/", string(os.PathSeparator))
```

Comment:
question: was it not possible before to run the tests on windows? I don't fully understand the scope of this change. Can you explain?